docs/source/_includes/create-genesis-block.inc

```diff
@@ -0,0 +1,215 @@
+        sawtooth.consensus.algorithm.name=PoET \
+        sawtooth.consensus.algorithm.version=0.1 \
+        sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/simulator_rk_pub.pem)" \
+        sawtooth.poet.valid_enclave_measurements=$(poet enclave measurement) \
+        sawtooth.poet.valid_enclave_basenames=$(poet enclave basename)
+
+   .. tip::
+
+      This is a complicated command. Here's an explanation of the options and
+      arguments:
+
+      ``--key /etc/sawtooth/keys/validator.priv``
+       Signs the proposal with this node's validator key. Only this key can be
+       used to change on-chain settings. For more information, see
+       :doc:`../sysadmin_guide/configuring_permissions`.
+
+      ``-o config-consensus.batch``
+       Wraps the consensus proposal transaction in a batch named
+       ``config-consensus.batch``.
+
+      ``sawtooth.consensus.algorithm.name``
+       Specifies the consensus algorithm for this network.
+
+      ``sawtooth.consensus.algorithm.version``
+       Specifies the version of the consensus algorithm.
+
+      (PBFT only) ``sawtooth.consensus.pbft.members``
+       Lists the member nodes on the initial network as a JSON-formatted
+       string of the validators' public keys, using the following format:
+
+       ``[<public-key-1>, <public-key-2>, ..., <public-key-n>]``
+
+      (PoET only) ``sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/simulator_rk_pub.pem)"``
+       Adds the public key for the PoET Validator Registry transaction
+       processor to use for the PoET simulator consensus.
+
+      (PoET only) ``sawtooth.poet.valid_enclave_measurements=$(poet enclave measurement)``
+       Adds a simulated enclave measurement to the blockchain. The
+       PoET Validator Registry transaction processor uses this value to check
+       signup information.
+
+      (PoET only) ``sawtooth.poet.valid_enclave_basenames=$(poet enclave basename)``
+       Adds a simulated enclave basename to the blockchain. The PoET
+       Validator Registry uses this value to check signup information.
+
+#. (PoET only) Create a batch to register the first Sawtooth node with the PoET
+   Validator Registry transaction processor. Without this command, the validator
+   would not be able to publish any blocks.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ poet registration create --key /etc/sawtooth/keys/validator.priv -o poet.batch
+
+#. (Optional) Create a batch to configure other consensus settings.
+
+   * For PBFT:
+
+     .. code-block:: console
+
+        [sawtooth@system]$ sawset proposal create \
+        --key /etc/sawtooth/keys/validator.priv \
+        -o pbft-settings.batch \
+        SETTING-NAME=VALUE \
+        ... \
+        SETTING-NAME=VALUE
+
+     For the available settings and their default values, see
+     `"On-Chain Settings" in the PBFT documentation
+     <https://sawtooth.hyperledger.org/docs/pbft/nightly/master/technical-information.html#on-chain-settings>`__.
```

Comment:
This links to a page that doesn't exist

Reply:
I changed the link to point to the PBFT doc list on the website's doc page (so that we don't have to go back and change the link once PBFT 1.0 has been released).